subprojects/snapshots/src/main/java/org/gradle/internal/fingerprint/FileCollectionFingerprint.java

```diff
@@ -39,25 +38,4 @@
 
     boolean wasCreatedWithStrategy(FingerprintingStrategy strategy);
```

Comment:
I wonder if the `wasCreatedWithStrategy` method now should go away and `DefaultCurrentFileCollectionFingerprint.from` should receive a `PreviousFileCollectionFingerprint` as candidate. Otherwise I think it doesn't make sense to expose the configuration hash here and have the `wasCreatedWithStrategy` method.